superchain/Dockerfile

```diff
@@ -235,7 +235,7 @@ COPY --from=bindist /opt/golang/go ${GOROOT}
 # Install Node (configurable with '--build-arg NODE_MAJOR_VERSION=xxx') and yarn
 # (Put this as late as possible in the Dockerfile so we get to reuse the layer cache
 # for most of the multiple builds).
-ARG NODE_MAJOR_VERSION="18"
+ARG NODE_MAJOR_VERSION="20"
```

Comment:
This isn't the fix you want. Let's sync offline.

Reply:
ok appreciate it